alicloud/service_alicloud_vpc.go

```diff
@@ -231,32 +231,67 @@ func (s *VpcService) QueryRouteTableById(routeTableId string) (rt vpc.RouteTable
 			return vpcClient.DescribeRouteTables(request)
 		})
 		if err != nil {
-			return err
+			return WrapErrorf(err, DefaultErrorMsg, "RouteTable", request.GetActionName(), AlibabaCloudSdkGoERROR)
```

Comment:
I think the method `QueryRouteTableById ` can be replaced by `DescribeRouteTable `.